### PR TITLE
SIDM-4643 - Allow azurerm_monitor_scheduled_query_rules_alert for idam-shared-infra

### PR DESCRIPTION
**Pull request by henrydobson**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/SIDM-4643

### Change description ###

Allow azurerm_monitor_scheduled_query_rules_alert resource type

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by henrydobson** [comment]
> use this branch https://github.com/hmcts/cnp-jenkins-config/blob/master/terraform-infra-approvals/global.json#L20

That branch does not work.

File: terraform-infra-approvals/idam-shared-infrastructure.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "resources": [],
   "module_calls": [
-    {"source":  "git@github.com:hmcts/cnp-module-waf?ref=SIDM-2128_add_idam_waf_exclusions"}
+    {"source":  "git@github.com:hmcts/cnp-module-waf?ref=SIDM-2128_add_idam_waf_exclusions"},
+    {"source":  "git@github.com:hmcts/cnp-module-key-vault?ref=change-to-sku-name"}
```
Comment (adusumillipraveen) [inline terraform-infra-approvals/idam-shared-infrastructure.json:5]: please use azurermv2 branch, its already whitelisted

Comment (henrydobson) [reply terraform-infra-approvals/idam-shared-infrastructure.json:5]: It doesn't work. https://github.com/hmcts/idam-shared-infrastructure/pull/50/commits/a73a7172112912fc360802253b5452c7298b11cd

**Comment by adusumillipraveen** [comment]
> > use this branch https://github.com/hmcts/cnp-jenkins-config/blob/master/terraform-infra-approvals/global.json#L20
> 
> That branch does not work. See [hmcts/idam-shared-infrastructure@a73a717](https://github.com/hmcts/idam-shared-infrastructure/commit/a73a7172112912fc360802253b5452c7298b11cd)



> > use this branch https://github.com/hmcts/cnp-jenkins-config/blob/master/terraform-infra-approvals/global.json#L20
> 
> That branch does not work. See [hmcts/idam-shared-infrastructure@a73a717](https://github.com/hmcts/idam-shared-infrastructure/commit/a73a7172112912fc360802253b5452c7298b11cd)

Looks like the build is deleted, can you retry again ? Cannot see why it's not working. It used in other places

**Comment by henrydobson** [comment]
> > That branch does not work. See [hmcts/idam-shared-infrastructure@a73a717](https://github.com/hmcts/idam-shared-infrastructure/commit/a73a7172112912fc360802253b5452c7298b11cd)
> 
> Looks like the build is deleted, can you retry again ? Cannot see why it's not working. It used in other places

https://sandbox-build.platform.hmcts.net/job/HMCTS_Sandbox_IDAM/job/idam-shared-infrastructure/view/change-requests/job/PR-50/48/console Failed.

**Pull request merged**